java-chassis-dependencies/pom.xml

```diff
@@ -32,7 +32,7 @@
 
   <properties>
     <jackson.version>2.9.4</jackson.version>
-    <vertx.version>3.5.0</vertx.version>
```

Comment:
Please update the License file for the update.
We need to update the NOTICE file as vertx 3.5.1 updated its NOTICE file.